src/foam/nanos/auth/Address.js

```diff
@@ -155,9 +155,7 @@ foam.CLASS({
           predicateFactory: function(e) {
             return e.OR(
               e.EQ(foam.nanos.auth.Address.STRUCTURED, false),
-              e.GTE(foam.mlang.StringLength.create({
-                arg1: foam.nanos.auth.Address.STREET_NUMBER
-              }), 1)
+              e.REG_EXP(foam.nanos.auth.Address.STREET_NUMBER, /^[0-9]{1,16}$/)
```

Comment:
House numbers/street numbers can consist of alphabet characters, surprisingly considering the number part of the noun.
Here's a bit of a blurb about it: https://www.architecturelab.net/whats-a-street-number-and-whats-an-address/

Reply:
@patrick ohh thank. I will fix it in another way

Reply:
Around here, we have 27A and B, and also 27 1/2  for example.  why are we bothering with this?

Reply:
in Contact.js there already checks Pattern streetNumber = Pattern.compile("^[0-9]{1,16}$"); but it is wrong. I will change to compare the length of input.